docs/core/tools/dotnet-nuget-why.md

```diff
@@ -0,0 +1,78 @@
+---
+title: dotnet nuget why command
+description: The 'dotnet nuget why' command provides the user to view the dependency graph of a package.
+ms.date: 08/04/2022
+---
+# dotnet nuget why
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+
+## Name
+
+`dotnet nuget why` - Prints the dependency graph for a package.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget why [<PROJECT>|<SOLUTION>] [<PACKAGE_NAME>]
+    [--framework <FRAMEWORK>]
+dotnet nuget why -h|--help
+```
+
+## Description
+
+The `dotnet list package` command provides a way to print out the dependency graph for a package to allow users to understand the nature of top-level packages and their transitive dependencies. This command is dependent on the `project.assets.json` file being present in the project. The following example shows the output of the `dotnet nuget why` command for `packageA` which has dependencies in `projectNameA` and `projectNameB`:
```

Comment:
This would be easier to understand with actual command output using real package/project names rather than packageA and B and projectA and B.